etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/LocalFileOrderSender.java

```diff
@@ -31,6 +35,7 @@ public void sendOrder(final Order<?> order) throws UnableToSendOrderException {
         logger.logInfo("Sending the order to the hard drive at {}", fileLocation.toAbsolutePath());
 
         try {
+            metaData.put(order.getFhirResourceId(), MetaDataStep.RECEIVED_FROM_REPORT_STREAM);
```

Comment:
Should this be `MetaDataStep.SENT_TO_REPORT_STREAM`?  Which even that doesn't fully make sense since we aren't sending to RS in this case.

Reply:
done.